pkgs/development/libraries/libfprint-tod/default.nix

```diff
@@ -5,33 +5,22 @@
 
 # for the curious, "tod" means "Touch OEM Drivers" meaning it can load
 # external .so's.
-libfprint.overrideAttrs ({ postPatch ? "", mesonFlags ? [], ... }: let
-  version = "1.90.7+git20210222+tod1";
-in  {
+libfprint.overrideAttrs (old: rec {
   pname = "libfprint-tod";
-  inherit version;
+  version = "1.94.7+tod1";
```

Comment:
Newer versions of libfprint+tod have problems with enrolling fingerprints and upstream refuses to fix this issue, so we can never update beyond `1.90.7+git20210222+tod1`.

https://github.com/NixOS/nixpkgs/issues/175212#issuecomment-1175119104
https://gitlab.freedesktop.org/libfprint/libfprint/-/issues/484

Reply:
That's unfortunate! I think what I'll do is create a second, version-pinned `libfprint-tod`, so we can continue updating it (`1.90.7+git20210222+tod1` is 3 years old at this point, and has at least one serious issue for me that's been fixed since— it breaks after a single fingerprint attempt rather than allowing me to try authenticating again), whilst still allowing users of buggy proprietary drivers (which the `libfrpint-tod` author understandably cannot fix) to use their hardware.

Reply:
Done this now. ^^